Jarvis-Platform/jarvis-studio
# Feature/profile page ui improvements

## Description
Profile page improvements with tabs & ui.


## Todos
- [x] Remove conf tab
- [x] Remove notes tab
- [x] Finish information tab
  - [x] Update display of accounts
  - [x] Remove Skype property
  - [x] Add space on left

## Deploy Notes
Review & merge


## Steps to Test or Reproduce
```sh
git checkout master
git pull
git checkout feature/profile-page-ui-improvements
```

1. Go to `/settings/profile` => Default tab should be `My info`
2. Click on `My configurations` => You should be able to access to your configurations listing
3. Click on `My notes` => You should be able to access to your notes + threads


## Impacted Areas in Application
List general components of the application that this PR will affect:

* `src/components/settings/profile`
